modules/applications/pages/ui-builder/how-to-configure-https.adoc

```diff
@@ -0,0 +1,141 @@
+= HTTPS Configuration
+:page-aliases: applications:how-to-configure-https.adoc
+:description: In production, it is necessary to set up both Bonita Runtime and UI Builder for HTTPS instead of HTTP. The easiest way it to configure Bonita UI Proxy for SSL offloading.
+
+{description}
+
+Configuring HTTPS for Bonita Runtime and UI Builder can be achieved just by updating the nginx configuration of Bonita UI proxy and setting some environment variables. This page contains some configuration  examples.
+
+== Prerequisites
+
+To configure HTTPS, you need to obtain a certificate issued by a certificate authority for our domain. It is also advised to create a Diffie-Hellman parameters file (dhparam.pem) for increased security.
+
+== Configure Bonita UI Proxy
+
+=== Configure the environment variables
+
+To activate HTTPS, you need to set the environment variable NGINX_LISTEN_ADDRESS in `docker-compose.yml` to the Bonita UI proxy service (`reverse-proxy`):
+
+[source,yaml]
+----
+services:
+  ...
+  reverse-proxy:
+    image: ...
+    environment:
+      NGINX_LISTEN_ADDRESS: 443 ssl
+----
+
+=== Volumes mount
+
+To configure HTTPS, you need to mount the certificate and key files into Bonita UI proxy container. You can do this by mounting volumes.
+You will also need other mounts to update the nginx configuration.
+
+Edit the `docker-compose.yml` file and add the following volumes/bind mounts to the reverse-proxy service (replacing <path_on_docker_host> by the local files paths):
+
+[source,yaml]
+----
+services:
+  ...
+  reverse-proxy:
+    image: ...
+    environment: ...
+    volumes:
+      - type: bind
+        source: <path_on_docker_host>/nginx.conf
+        target: /etc/nginx/templates/nginx.conf.template
+      - type: bind
+        source: <path_on_docker_host>/https.conf
+        target: /etc/nginx/snippets/https.conf
+      - <path_on_docker_host>/nginx.crt:/etc/ssl/certs/nginx.crt
+      - <path_on_docker_host>/nginx.key:/etc/ssl/private/nginx.key
+      - <path_on_docker_host>/dhparam.pem:/etc/ssl/certs/dhparam.pem
+----
+
+=== Update the nginx configuration to activate HTTPS
+
+Start the service so that `nginx.conf` file is create on the Docker host, and update it to include a conf snippet to the server block:
```

Comment:
```suggestion
Start the service so that the `nginx.conf` file is creates on the Docker host. Then, update it to include a conf snippet to the server block:
```